src/getting-started.md

```diff
@@ -33,71 +33,65 @@ instead.
 
 ### Cards
 
-A question and answer pair is called a 'card'. This is based on a paper
-flashcard with a question on one side and the answer on the back. In
-Anki a card doesn’t actually look like a physical card, and when you
-show the answer the question remains visible by default. For example, if
-you’re studying basic chemistry, you might see a question like:
+A question and answer pair is called a _card_. It's similar to a paper
+flashcard with a question on front and answer on the back. However, in
+Anki, a card doesn't actually look like a physical card, and when you
+look at the answer the question remains visible by default. For example, if
+you're studying basic chemistry, you might see a question like:
 
     Q: Chemical symbol for oxygen?
 
-After thinking about it, and deciding the answer is O, you click the
-show answer button, and Anki shows you:
+After deciding the answer is O, you click the
+"Show Answer" button, and Anki shows you:
 
     Q: Chemical symbol for oxygen?
     A: O
 
-After confirming that you are correct, you can tell Anki how well you
-remembered, and Anki will choose a next time to show you again.
+After confirming that you are correct, you tell Anki how well you
+remembered the answer, and Anki will choose when to show you the card again. If, for example, Anki decides to show you the card again in 30 days, we say the card has a 30 day interval.
```

Comment:
```suggestion
remembered the answer, and Anki will choose when to show you the card again. For example, if Anki decides to show you the card again in 3 days, we say the card has a 3 day interval.
```
30d will seem like an eternity to a new user. I picked 3 arbitrarily, but anything 2-5 would probably be just as good.